articles/osquery-evented-tables-overview.md

```diff
@@ -0,0 +1,162 @@
+
+Due to the data volume, Fleet leans towards using `file_events` for macOS, but does not recommend against using `es_process_file_events`.
+
+#### FIM on Linux
+To turn `file_events` on Linux, use the flag `--enable_file_events=true`. The corresponding utility is [inotify](https://man7.org/linux/man-pages/man7/inotify.7.html). 
+
+Linux has a `process_file_events` table which uses the [audit framework](https://wiki.archlinux.org/title/Audit_framework). To use this table, use the flags `--disable_audit=false --audit_allow_fim_events=true`. 
+
+Fleet recommends using the `process_file_events` table since it also includes data for which process accessed which file. 
+
+#### FIM on Windows
+On Windows, use the `--enable_ntfs_event_publisher=true` flag to turn on `ntfs_journal_events`. The corresponding utility is [NTFS Journal](https://docs.microsoft.com/en-us/windows/win32/fileio/change-journal-records). 
+
+#### Learn more:
+* [Osquery FIM docs](https://osquery.readthedocs.io/en/stable/deployment/file-integrity-monitoring/)
+
+### Process auditing
+Process auditing refers to recording process executions and network, or socket, connections. 
+
+#### Process auditing on Linux
+On Linux, there are two utilities that enable osquery process auditing: [eBPF](https://ebpf.io/what-is-ebpf) and the [audit framework](https://wiki.archlinux.org/title/Audit_framework). 
+
+The choice of utility depends on your situation. Some relevant considerations are:
+- Audit has earlier support (>2.6 ) compared to eBPF (>4.18)
+- Only one consumer of audit’s logs are allowed at a time. The `--audit_persist=true` flag will set osquery to retry connection to audit logs.
+- Audit has limited visibility inside containers
+- The audit table and the eBPF table return slightly different data
+
+Overall, Fleet recommends you try both and compare results for your use case.
+
+To use the `bpf_process_events` and `bpf_socket_events` tables, use the flag `--enable_bpf_events=true`. See [these instructions](https://osquery.readthedocs.io/en/latest/deployment/process-auditing/#linux-process-and-socket-auditing-using-bpf) for more information. 
+
+To use `process_events` and `socket_events` with the audit framework, use the flags `--disable_audit=false --audit_allow_process_events=true --audit_allow_socket_events=true`. See [these instructions](https://osquery.readthedocs.io/en/latest/deployment/process-auditing/#linux-process-auditing-using-audit) for more information. 
+
+#### Process auditing on macOS
+On macOS, there are two utilities that enable osquery process auditing: [OpenBSM](https://github.com/openbsm/openbsm) and the [EndpointSecurity](https://developer.apple.com/documentation/endpointsecurity). Fleet recommends using the EndpointSecurity implementation because it is intended to replace OpenBSM. EndpointSecurity is available starting macOS 10.15.
```

Comment:
If you think it's necessary, we can also explicitly mention here that Apple has deprecated OpenBSM and it is slated for removal in some future macOS version